persistence/src/main/java/edu/unc/lib/dl/persist/services/delete/MarkForDeletionJob.java

```diff
@@ -62,6 +63,11 @@ public void run() {
 
         RepositoryObject repoObj = repositoryObjectLoader.getRepositoryObject(pid);
 
+        if (repoObj instanceof AdminUnit) {
+            aclService.assertHasAccess("Insufficient privileges to delete/restore admin unit " + pid.getUUID(),
```

Comment:
since there are separate jobs for restore and delete, this assertion should just say "delete"